batch/ilastik_wrapper.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+"""ilastik wrapper script for HPC server."""
+
+import json
+import logging
+import os
+import re
+import subprocess
+import time
+from pathlib import Path
+from urllib.parse import urljoin
+
+import requests
+
+logging.basicConfig(format="%(asctime)-30s %(filename)-20s %(levelname)-10s %(message)s", level=logging.INFO)
+logging.Formatter.converter = time.gmtime
+logging.Formatter.default_time_format = "%Y-%m-%dT%H:%M:%S"
+logging.Formatter.default_msec_format = "%s.%03dZ"
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    os.environ["PATH"] = os.pathsep.join(
+        [
+            os.path.expanduser("~/miniconda3/envs/ildev/bin"),
```

Comment:
Should this be configured somehow?

Reply:
This was copied verbatim from the existing script, but yes, I also don't like it.

Anyway, it's probably better not to copy scripts with the payload and config every time someone submits a job. We should embed this in a deployed environment somehow (which will probably include the whole Python + deps).